functions/src/utils.js

```diff
@@ -49,3 +49,10 @@ exports.typesenseDocumentFromSnapshot = async (
 
   return typesenseDocument;
 };
+
+exports.importDocumentsConfig = (batchSize) => ({
+  action: "emplace",
+  dirty_values: "coerce_or_drop",
+  return_doc: false,
```

Comment:
This is a good improvement, I've seen users miss the error returned because we log the document as well. 

But could you instead set `return_id: true`, so at least the document ID is logged for users to investigate further?